android/src/main/kotlin/com/airship/flutter/AirshipPlugin.kt

```diff
@@ -354,7 +394,10 @@ class AirshipPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
         result.success(null)
     }
 
-    private fun applyTagGroupOperations(editor: TagGroupsEditor, operations: ArrayList<Map<String, Any?>>) {
+    private fun applyTagGroupOperations(
```

Comment:
It would be great to just do a commit with lint stuff applied so we can stop having it distract from the actual changes

Reply:
true